--- ./work/glibc-2.11.1/nptl/sysdeps/pthread/pt-initfini.c        2009-12-08 20:10:20.000000000 +0000
+++ ./work/glibc-2.11.1/nptl/sysdeps/pthread/pt-initfini.c.new    2019-09-26 17:00:18.263289072 +0000
@@ -45,6 +45,11 @@
 /* Embed an #include to pull in the alignment and .end directives. */
 asm ("\n#include \"defs.h\"");
 
+asm ("\n#if defined __i686 && defined __ASSEMBLER__");
+asm ("\n#undef __i686");
+asm ("\n#define __i686 __i686");
+asm ("\n#endif");
+
 /* The initial common code ends here. */
 asm ("\n/*@HEADER_ENDS*/");
